frontend/packages/cli/src/cli/runPreprocess.ts

```diff
@@ -26,7 +26,13 @@ export async function runPreprocess(
 
   let json = null
   try {
-    json = await parse(input, format)
+    const { value, errors } = await parse(input, format)
+    json = value
+    if (errors.length > 0) {
+      for (const error of errors) {
+        console.error(error)
+      }
+    }
```

Comment:
Eventually, we would like to remove try-catch altogether, but this will be addressed in a separate PR.